update_changelog.sh

```diff
@@ -16,7 +16,7 @@ fi
 # Remove the header so we can append the additions
 tail -n +3 "$CHANGELOG_FILE" > "$CHANGELOG_FILE.tmp" && mv "$CHANGELOG_FILE.tmp" "$CHANGELOG_FILE"
 
-docker run -it --rm -v "$(pwd)":/usr/local/src/your-app ferrarimarco/github-changelog-generator -u stargate -p stargate -t $GITHUB_TOKEN --since-tag $previous_version -b $CHANGELOG_FILE
+docker run -it --rm -v "$(pwd)":/usr/local/src/your-app ferrarimarco/github-changelog-generator -u stargate -p stargate -t $GITHUB_TOKEN --since-tag $previous_version --base $CHANGELOG_FILE --output $CHANGELOG_FILE --release-branch 'main' --exclude-tags-regex 'v1.*' --exclude-labels 'stargate-v1,duplicate,question,invalid,wontfix'
```

Comment:
Since we are merging main to v1, maybe `--release-branch 'main'` should be omitted.. We would include the fixes on v1 in v2 release notes, which makes sense imo.. But now sure..